pages/pil/dashboard/index.js

```diff
@@ -91,18 +91,22 @@ module.exports = settings => {
       res.locals.static.isAsru = req.user.profile.asruUser;
       res.locals.static.isLicensing = req.user.profile.asruLicensing;
 
-      // can only transfer own pil if it's active and has no in-progress amendment
-      const hasOpenAmendment = req.pil.status === 'active' && get(req.model, 'openTasks[0].data.action') === 'grant';
-      res.locals.static.canTransferPil = req.pil.status === 'active' && req.user.profile.id === req.profile.id && !hasOpenAmendment;
+      return req.user.can('pil.transfer', { pilId: req.pilId })
+        .then(canTransfer => {
+          const hasOpenAmendment = req.pil.status === 'active' && get(req.model, 'openTasks[0].data.action') === 'grant';
 
-      next();
+          res.locals.static.canTransferPil = req.pil.status === 'active' && !hasOpenAmendment && canTransfer;
+        })
+        .then(() => next())
+        .catch(next);
     }
   }));
 
   app.post('/', redirectToTaskIfOpen());
 
   app.post('/', (req, res, next) => {
     sendData(req)
+      .then(() => delete req.session.form[req.model.id])
```

Comment:
What's this for?

Reply:
We were not clearing the form after submission so if you went back into it it was persisting the values

Reply:
👍